src/feedbackMessages.js

```diff
@@ -0,0 +1,16 @@
+const Language = require("./config/language.js");
+
+const MESSAGE = {
```

Comment:
In the spirit of maintaining the style applied throughout the yorlang source code, please use the lower camel pattern for your object names ```message```